common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/ExecutorDiskUtils.java

```diff
@@ -40,7 +40,7 @@ public static File getFile(String[] localDirs, int subDirsPerLocalDir, String fi
     // So we are creating a File just to get the normalized path back to intern it.
     // Finally a new File is built and returned with this interned normalized path.
```

Comment:
Update the comment so it won't cause confusion.
"Finally a new File is built and returned with this interned normalized path." -> "We return this interned normalized path."